README.md

```diff
@@ -817,6 +818,13 @@ A [news aggregator](https://en.wikipedia.org/wiki/News_aggregator), also termed
 - [Winds](https://getstream.io/winds/) `⚠` - Open source and beautiful RSS reader built using React/Redux/Sails/Node and Stream. It showcases personalized feeds powered by the Stream API. ([Demo](https://winds.getstream.io/), [Source Code](https://github.com/GetStream/Winds)) `BSD-3-Clause` `Nodejs`
 
 
+### File Encryption
+
+**[`^        back to top        ^`](#)**
+
+- [Hat.sh](https://hat.sh/) - In-browser Secure Local File Encryption. ([Demo](https://hat.sh/), [Source Code](https://github.com/sh-dv/hat.sh))  `MIT` `JavaScript`
```

Comment:
Any category you are creating has the minimum requirement of 3 items.
If not, your addition may be inserted into Misc/Other.